testutil/testmigration/fixtures.go

```diff
@@ -0,0 +1,130 @@
+package testmigration
+
+import (
+	"encoding/binary"
+	"fmt"
+	"math/rand"
+
+	cometcrypto "github.com/cometbft/cometbft/crypto/ed25519"
+	cmtjson "github.com/cometbft/cometbft/libs/json"
+	cosmostypes "github.com/cosmos/cosmos-sdk/types"
+	"github.com/regen-network/gocuke"
+	"github.com/stretchr/testify/require"
+
+	"github.com/pokt-network/poktroll/app/volatile"
+	migrationtypes "github.com/pokt-network/poktroll/x/migration/types"
+)
+
+// NewMorseStateExportAndAccountStateBytes returns:
+// - A serialized MorseStateExport
+// - Its corresponding MorseAccountState
+//
+// The states are populated with:
+// - Random account addresses
+// - Monotonically increasing balances/stakes
+// - One application per account
+// - One supplier per account
+//
+// TODO_CONSIDERATION: Test/benchmark execution speed can be optimized by refactoring this to a pre-generate fixture.
+func NewMorseStateExportAndAccountStateBytes(
+	t gocuke.TestingT,
+	numAccounts int,
+) (morseStateExportBz []byte, morseAccountStateBz []byte) {
+	morseStateExport, morseAccountState := NewMorseStateExportAndAccountState(t, numAccounts)
+
+	var err error
+	morseStateExportBz, err = cmtjson.Marshal(morseStateExport)
+	require.NoError(t, err)
+
+	morseAccountStateBz, err = cmtjson.Marshal(morseAccountState)
+	require.NoError(t, err)
+
+	return morseStateExportBz, morseAccountStateBz
+}
+
+// NewMorseStateExportAndAccountState returns MorseStateExport and MorseAccountState
```

Comment:
Can you #PUC the difference between State & Account here (two bullet points are fine)